packages/angular-material/test/boolean-control.spec.ts

```diff
@@ -96,23 +160,181 @@ describe('Boolean control', () => {
             }
         });
         mockSubStore.complete();
+        fixture.detectChanges();
         component.ngOnInit();
-        MockNgRedux.reset();
-        const mockSubStore2 = MockNgRedux.getSelectorStub();
-        mockSubStore2.next({
+
+        expect(component.data).toBe(true);
+        expect(checkboxInstance.checked).toBe(true);
```

Comment:
I don't think we necessarily have to expect these (here and in the following tests) as we already covered that case

Reply:
yes, we could leave them out